assets/info-page/index.html

```diff
@@ -36,6 +54,136 @@
       }
```

Comment:
Can't comment on line above, but looks like after we select, it closes the window. Do we tell the user to switch back to Mattermost at any point? Does this work on Desktop ok?

Reply:
My understanding is that this flow has been already tested.

Since my task here was more about the design, I haven't properly tested the whole flow. I would need to setup a Teams instance, with everything that it entails, configure the plugin, and then do the login. Not sure if I am able to do that. If you want me to retest this, let's talk to see how can we set this up.

There are no explicit instructions to get back to Mattermost. The normal flow should be that the window closes, and the window you have behind is where you navigated to this page from (mattermost web or desktop).

We could add the "You can close this window now" message again when the setting is set, and they know they have to close and go back to Mattermost, but 0/5.

Reply:
Sounds good, this is something we'll revisit after merging. Thank you for adding clarity!